src/region/RegionAnnotations.tsx

```diff
@@ -41,22 +35,22 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
 
     state: State = {};
 
+    componentDidUpdate(_prevProps: Props, prevState: State): void {
```

Comment:
Do we need the trailing underscore in front of `prevProps`?

Reply:
It indicates that the argument is knowingly not used for eslint: https://eslint.org/docs/rules/no-unused-vars#argsignorepattern